src/App.js

```diff
@@ -1,86 +1,124 @@
-import React from 'react';
+import React, { useState, useEffect, useMemo } from 'react';
+import { Header } from './components/Header/Header';
+import { TodoList } from './components/TodoList/TodoList';
+import { Filter } from './components/Filter/Filter';
+
+const filters = {
+  all: 'all',
+  active: 'active',
+  completed: 'completed',
+};
```

Comment:
For constants we use uppercase names: e.g:
```
const filters = {
  ALL: 'all',
  ACTIVE: 'active',
  COMPLETED: 'completed',
};
```

And let's move it to a separate file and import it where we need it